charts/osm/templates/fluentbit-configmap.yaml

```diff
@@ -21,10 +21,10 @@ data:
       match      *
       regex       message /"level":"error"/
     [OUTPUT]
-      Name    {{ .Values.OpenServiceMesh.fluentBit.defaultOutput }}
+      Name    {{ .Values.OpenServiceMesh.fluentBit.outputName }}
       Match   *
-      {{- if not .Values.OpenServiceMesh.fluentBit.allowCustomOutput }}
-      Customer_ID {{ .Values.OpenServiceMesh.fluentBit.workspaceID }}
+      {{- if eq .Values.OpenServiceMesh.fluentBit.outputName "azure" "Azure" }}
```

Comment:
if `Azure` also an acceptable value for `Name`?

Reply:
yep, either works